src/NuGet.Indexing/Retry.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Is it fair to assume this change won't break any other high level dependencies/not used at all? or we are going to treat this as a breaking change?

Reply:
`NuGet.Indexing` depends on `NuGet.Services.Metadata.Catalog` so the class is still available to the previous users.